composer.json

```diff
@@ -1,85 +1,100 @@
 {
-  "name": "phalcon/cphalcon",
-  "description": "Phalcon is an open source web framework delivered as a C extension for the PHP language providing high performance and lower resource consumption.",
-  "keywords": [
-    "extension",
-    "phalcon",
-    "framework",
-    "high load",
-    "mvc",
-    "psr-7",
-    "psr-17"
-  ],
-  "license": "BSD-3-Clause",
-  "authors": [
-    {
-      "name": "Phalcon Team",
-      "email": "team@phalcon.io",
-      "homepage": "https://phalcon.io/en/team"
+    "name": "phalcon/cphalcon",
```

Comment:
Could you do not change this file in this PR? Can we change it separately?

Reply:
Not sure what you exactly mean

Reply:
I mean changing indents should be consistent with this
https://github.com/phalcon/cphalcon/blob/ef31ca8fdfdcbaeef24bd93a32107a43b6c1ba6b/.editorconfig#L25-L26

Reply:
So, just add what you need w/o changing indents, because it is unrelated to this PR

Reply:
Ah sorry. Must be some auto update. Moved it back to 2